documentation/modules/exploit/multi/iiop/cve_2023_21839_weblogic_rce.md

```diff
@@ -0,0 +1,120 @@
+## Vulnerable Application
+
+### Description
+Oracle Weblogic 12.2.1.3.0, 12.2.1.4.0 and 14.1.1.0.0 prior to the Jan 2023 security update are vulnerable to an unauthenticated
+remote code execution vulnerability due to a post deserialization vulnerability. This occurs when an attacker serializes
+a `ForeignOpaqueReference` class object, deserializes it on the target, and then post deserialization, calls the
+object's `getReferent()` method, which will make use of the `ForeignOpaqueReference` class's `remoteJNDIName` variable,
```

Comment:
No getReferent is not a typo just in case anyone reviewing this is curious. That is legitimately how its spelled 😕